src/Nethermind/Nethermind.Serialization.Rlp/WithdrawalDecoder.cs

```diff
@@ -23,7 +23,7 @@ public class WithdrawalDecoder : IRlpStreamDecoder<Withdrawal>, IRlpValueDecoder
             Index = rlpStream.DecodeULong(),
             ValidatorIndex = rlpStream.DecodeULong(),
             Address = rlpStream.DecodeAddress(),
-            Amount = rlpStream.DecodeUInt256()
+            AmountInGWei = rlpStream.DecodeUlong()
```

Comment:
Different methods when reading the same `ulong` value: `DecodeUlong` vs `DecodeULong`. This should be addressed to avoid any confusion in the future.

Reply:
I've just used the method that we have now :) Let's rename it in separate PR